addon/components/ui-rating.js

```diff
@@ -1,16 +1,29 @@
 import Ember from 'ember';
 import Base from '../mixins/base';
 
-var Rating = Ember.Component.extend(Base,{
+var Rating = Ember.Component.extend(Base, {
   module: 'rating',
   classNames: [ 'ui', 'rating' ],
 
-  init: function() {
-    this._super();
-
+  init() {
+    this._super(...arguments);
     if (!this.get('initialRating') && this.get('rating')) {
       this.set('initialRating', this.get('rating'));
     }
+  },
+
+  didInsertElement() {
+    this.set('initialized', false);
```

Comment:
This is a little bit of a hack, probably could add a pull request to the main semantic repository to ignore calling initial values like on checkbox.